functions.py

```
@@ -0,0 +1,203 @@
+    new_df = pl.concat([no_pk_df, pk50_df, pk90_df])
+
+    return new_df
+
+
+def dataset_dropout(dataset_name, train_indices, dataset2drop):
+
+    # dataset_name=pl.Series(dataset_name)
+    dataset_filter = pl.Series(dataset_name).str.starts_with(dataset2drop)
+    dataset_filter = dataset_filter.to_numpy()
+
+    dropout_indcies = set(np.where(dataset_filter == False)[0])
```

Ruff (0.8.2)

Local variable `dropout_indcies` is assigned to but never used

Remove assignment to unused variable `dropout_indcies`

(F841)